!SESSION 2019-04-16 21:58:20.153 -----------------------------------------------
eclipse.buildId=4.3.2.M20140221-1700
java.version=1.8.0_05
java.vendor=Oracle Corporation
BootLoader constants: OS=win32, ARCH=x86_64, WS=win32, NL=en_US
Framework arguments:  -product org.eclipse.epp.package.cpp.product -pluginCustomization C:/intelfpga/17.0/nios2eds/bin/eclipse_nios2/plugin_customization.ini
Command-line arguments:  -os win32 -ws win32 -arch x86_64 -product org.eclipse.epp.package.cpp.product -pluginCustomization C:/intelfpga/17.0/nios2eds/bin/eclipse_nios2/plugin_customization.ini

!ENTRY org.eclipse.cdt.core 1 0 2019-04-16 22:12:14.871
!MESSAGE Indexed 'nios_system_bsp' (80 sources, 62 headers) in 1.93 sec: 1,190 declarations; 2,486 references; 110 unresolved inclusions; 6 syntax errors; 191 unresolved names (4.94%)

!ENTRY org.eclipse.cdt.core 1 0 2019-04-16 22:12:17.611
!MESSAGE Indexed 'nios_system' (0 sources, 0 headers) in 0.00 sec: 0 declarations; 0 references; 0 unresolved inclusions; 0 syntax errors; 0 unresolved names (0.00%)

!ENTRY org.eclipse.jface 2 0 2019-04-16 22:29:44.658
!MESSAGE Keybinding conflicts occurred.  They may interfere with normal accelerator operation.
!SUBENTRY 1 org.eclipse.jface 2 0 2019-04-16 22:29:44.658
!MESSAGE A conflict occurred for F3:
Binding(F3,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.edit.opendecl,Open Declaration,
		Open an editor on the selected element's declaration(s),
		Category(org.eclipse.cdt.ui.category.source,Source,Source commands,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@2106b1f0,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cEditorScope,,,system)
Binding(F3,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.edit.opendecl,Open Declaration,
		Open an editor on the selected element's declaration(s),
		Category(org.eclipse.cdt.ui.category.source,Source,Source commands,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@2106b1f0,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cViewScope,,,system)
!SUBENTRY 1 org.eclipse.jface 2 0 2019-04-16 22:29:44.658
!MESSAGE A conflict occurred for CTRL+SHIFT+T:
Binding(CTRL+SHIFT+T,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.navigate.opentype,Open Element,
		Open an element in an Editor,
		Category(org.eclipse.cdt.ui.category.source,Source,Source commands,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@11743654,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cEditorScope,,,system)
Binding(CTRL+SHIFT+T,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.navigate.opentype,Open Element,
		Open an element in an Editor,
		Category(org.eclipse.cdt.ui.category.source,Source,Source commands,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@11743654,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cViewScope,,,system)
!SUBENTRY 1 org.eclipse.jface 2 0 2019-04-16 22:29:44.658
!MESSAGE A conflict occurred for F4:
Binding(F4,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.edit.open.type.hierarchy,Open Type Hierarchy,
		Open a type hierarchy on the selected element,
		Category(org.eclipse.ui.category.navigate,Navigate,null,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@6c3dae0f,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cEditorScope,,,system)
Binding(F4,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.edit.open.type.hierarchy,Open Type Hierarchy,
		Open a type hierarchy on the selected element,
		Category(org.eclipse.ui.category.navigate,Navigate,null,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@6c3dae0f,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cViewScope,,,system)
!SUBENTRY 1 org.eclipse.jface 2 0 2019-04-16 22:29:44.658
!MESSAGE A conflict occurred for ALT+CTRL+I:
Binding(ALT+CTRL+I,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.edit.open.include.browser,Open Include Browser,
		Open an include browser on the selected element,
		Category(org.eclipse.ui.category.navigate,Navigate,null,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@17993651,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cEditorScope,,,system)
Binding(ALT+CTRL+I,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.edit.open.include.browser,Open Include Browser,
		Open an include browser on the selected element,
		Category(org.eclipse.ui.category.navigate,Navigate,null,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@17993651,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cViewScope,,,system)
!SUBENTRY 1 org.eclipse.jface 2 0 2019-04-16 22:29:44.658
!MESSAGE A conflict occurred for ALT+SHIFT+R:
Binding(ALT+SHIFT+R,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.edit.text.rename.element,Rename - Refactoring ,
		Rename the selected element,
		Category(org.eclipse.cdt.ui.category.refactoring,Refactor - C++,C/C++ Refactorings,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@3848c63,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cEditorScope,,,system)
Binding(ALT+SHIFT+R,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.edit.text.rename.element,Rename - Refactoring ,
		Rename the selected element,
		Category(org.eclipse.cdt.ui.category.refactoring,Refactor - C++,C/C++ Refactorings,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@3848c63,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cViewScope,,,system)
!SUBENTRY 1 org.eclipse.jface 2 0 2019-04-16 22:29:44.658
!MESSAGE A conflict occurred for CTRL+SHIFT+G:
Binding(CTRL+SHIFT+G,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.search.findrefs,References,
		Search for references to the selected element in the workspace,
		Category(org.eclipse.cdt.ui.category.source,Source,Source commands,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@33f2ccd1,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cEditorScope,,,system)
Binding(CTRL+SHIFT+G,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.search.findrefs,References,
		Search for references to the selected element in the workspace,
		Category(org.eclipse.cdt.ui.category.source,Source,Source commands,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@33f2ccd1,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cViewScope,,,system)
!SUBENTRY 1 org.eclipse.jface 2 0 2019-04-16 22:29:44.658
!MESSAGE A conflict occurred for CTRL+G:
Binding(CTRL+G,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.search.finddecl,Declaration,
		Search for declarations of the selected element in the workspace,
		Category(org.eclipse.cdt.ui.category.source,Source,Source commands,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@323e5176,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cEditorScope,,,system)
Binding(CTRL+G,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.search.finddecl,Declaration,
		Search for declarations of the selected element in the workspace,
		Category(org.eclipse.cdt.ui.category.source,Source,Source commands,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@323e5176,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cViewScope,,,system)
!SUBENTRY 1 org.eclipse.jface 2 0 2019-04-16 22:29:44.658
!MESSAGE A conflict occurred for ALT+CTRL+H:
Binding(ALT+CTRL+H,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.edit.open.call.hierarchy,Open Call Hierarchy,
		Open the call hierarchy for the selected element,
		Category(org.eclipse.ui.category.navigate,Navigate,null,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@6ccbd2de,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cEditorScope,,,system)
Binding(ALT+CTRL+H,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.edit.open.call.hierarchy,Open Call Hierarchy,
		Open the call hierarchy for the selected element,
		Category(org.eclipse.ui.category.navigate,Navigate,null,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@6ccbd2de,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cViewScope,,,system)
!SUBENTRY 1 org.eclipse.jface 2 0 2019-04-16 22:29:44.658
!MESSAGE A conflict occurred for CTRL+SHIFT+H:
Binding(CTRL+SHIFT+H,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.navigate.open.type.in.hierarchy,Open Type in Hierarchy,
		Open a type in the type hierarchy view,
		Category(org.eclipse.ui.category.navigate,Navigate,null,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@4b8bacb0,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cEditorScope,,,system)
Binding(CTRL+SHIFT+H,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.navigate.open.type.in.hierarchy,Open Type in Hierarchy,
		Open a type in the type hierarchy view,
		Category(org.eclipse.ui.category.navigate,Navigate,null,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@4b8bacb0,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cViewScope,,,system)

!ENTRY com.altera.sbtgui.launch 1 0 2019-04-16 22:41:33.013
!MESSAGE Executing: [C:/intelfpga/17.0/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=DE-SoC on localhost [USB-1]' --device=2 --instance=0 --sidp=0x1810 --id=0xad --timestamp=1555467743 /cygdrive/s/CPET-561/Arbitration_Demo/Part_1/HW/software/nios_system/nios_system.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2019-04-16 22:41:37.662
!MESSAGE Executing: [C:/intelfpga/17.0/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=DE-SoC on localhost [USB-1]' --device=2 --instance=0 --tcpport=auto --sidp=0x1810 --id=0xad --timestamp=1555467743]

!ENTRY org.eclipse.jface 2 0 2019-04-16 22:42:27.094
!MESSAGE Keybinding conflicts occurred.  They may interfere with normal accelerator operation.
!SUBENTRY 1 org.eclipse.jface 2 0 2019-04-16 22:42:27.094
!MESSAGE A conflict occurred for CTRL+T:
Binding(CTRL+T,
	ParameterizedCommand(Command(org.eclipse.cdt.ui.edit.open.quick.type.hierarchy,Quick Type Hierarchy,
		Shows quick type hierarchy,
		Category(org.eclipse.cdt.ui.category.source,Source,Source commands,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@7b16def6,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.cdt.ui.cEditorScope,,,system)
Binding(CTRL+T,
	ParameterizedCommand(Command(org.eclipse.debug.ui.commands.toggleMemoryMonitorsPane,Toggle Memory Monitors Pane,
		Toggle visibility of the Memory Monitors Pane,
		Category(org.eclipse.debug.ui.category.run,Run/Debug,Run/Debug command category,true),
		org.eclipse.ui.internal.WorkbenchHandlerServiceHandler@463f7e2f,
		,,true),null),
	org.eclipse.ui.defaultAcceleratorConfiguration,
	org.eclipse.debug.ui.memoryview,,,system)

!ENTRY com.altera.sbtgui.launch 1 0 2019-04-16 22:44:20.334
!MESSAGE Executing: [C:/intelfpga/17.0/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=DE-SoC on localhost [USB-1]' --device=2 --instance=0 --sidp=0x1810 --id=0xad --timestamp=1555467743 /cygdrive/s/CPET-561/Arbitration_Demo/Part_1/HW/software/nios_system/nios_system.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2019-04-16 22:44:25.166
!MESSAGE Executing: [C:/intelfpga/17.0/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=DE-SoC on localhost [USB-1]' --device=2 --instance=0 --tcpport=auto --sidp=0x1810 --id=0xad --timestamp=1555467743]

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:50:51.318
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Database.markFileIncomplete(Database.java:775)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:738)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.markFileIncomplete(Database.java:773)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:50:51.318
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.markFileIncomplete(Database.java:773)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:738)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:50:51.338
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:50:51.338
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:50:51.348
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:50:51.348
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:50:51.348
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 9 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:50:51.348
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:51:19.988
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Database.markFileIncomplete(Database.java:775)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:738)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.markFileIncomplete(Database.java:773)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:51:19.988
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.markFileIncomplete(Database.java:773)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:738)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:51:19.988
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:51:19.988
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:51:19.988
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 9 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:51:19.988
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:51:19.998
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:51:19.998
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:51:20.038
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:51:20.038
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:51:20.048
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:51:20.048
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:51:20.048
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 9 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:51:20.048
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY com.altera.sbtgui.launch 1 0 2019-04-16 22:51:22.079
!MESSAGE Executing: [C:/intelfpga/17.0/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=DE-SoC on localhost [USB-1]' --device=2 --instance=0 --sidp=0x1810 --id=0xad --timestamp=1555467743 /cygdrive/s/CPET-561/Arbitration_Demo/Part_1/HW/software/nios_system/nios_system.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2019-04-16 22:51:27.021
!MESSAGE Executing: [C:/intelfpga/17.0/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=DE-SoC on localhost [USB-1]' --device=2 --instance=0 --tcpport=auto --sidp=0x1810 --id=0xad --timestamp=1555467743]

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:54:19.478
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:54:19.478
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:54:19.548
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:54:19.548
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:54:19.558
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:54:19.558
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:54:19.558
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:54:19.558
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 22:54:19.568
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 9 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 22:54:19.568
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY com.altera.sbtgui.launch 1 0 2019-04-16 22:56:40.028
!MESSAGE Executing: [C:/intelfpga/17.0/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=DE-SoC on localhost [USB-1]' --device=2 --instance=0 --sidp=0x1810 --id=0xad --timestamp=1555467743 /cygdrive/s/CPET-561/Arbitration_Demo/Part_1/HW/software/nios_system/nios_system.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2019-04-16 22:56:44.642
!MESSAGE Executing: [C:/intelfpga/17.0/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=DE-SoC on localhost [USB-1]' --device=2 --instance=0 --tcpport=auto --sidp=0x1810 --id=0xad --timestamp=1555467743]

!ENTRY com.altera.sbtgui.launch 1 0 2019-04-16 23:00:42.658
!MESSAGE Executing: [C:/intelfpga/17.0/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=DE-SoC on localhost [USB-1]' --device=2 --instance=0 --sidp=0x1810 --id=0xad --timestamp=1555467743 /cygdrive/s/CPET-561/Arbitration_Demo/Part_1/HW/software/nios_system/nios_system.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2019-04-16 23:00:47.285
!MESSAGE Executing: [C:/intelfpga/17.0/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=DE-SoC on localhost [USB-1]' --device=2 --instance=0 --tcpport=auto --sidp=0x1810 --id=0xad --timestamp=1555467743]

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:02:27.527
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:02:27.527
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:02:27.567
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:02:27.567
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:02:27.577
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:02:27.577
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:02:27.587
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 9 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:02:27.587
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:03:34.867
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:03:34.867
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:03:34.867
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:03:34.867
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:03:34.867
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 9 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:03:34.867
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:03:34.877
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:03:34.877
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:03:35.007
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:03:35.007
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:03:35.007
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:03:35.007
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:03:35.017
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:03:35.017
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:03:35.017
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 9 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:03:35.017
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.528
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.528
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.528
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.528
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.528
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 9 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.528
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.538
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.538
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:527)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.558
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.558
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.parseLinkage(AbstractIndexerTask.java:923)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:537)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.558
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 10 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.558
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.giveUpExclusiveLock(Database.java:700)
	at org.eclipse.cdt.internal.core.pdom.PDOM.releaseWriteLock(PDOM.java:1042)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:141)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.releaseWriteLock(WritableCIndex.java:126)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.setResume(AbstractIndexerTask.java:609)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:587)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.558
!MESSAGE Error: IOException
!STACK 1
org.eclipse.core.runtime.CoreException: IOException
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:54)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)
Caused by: java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	... 9 more
!SUBENTRY 1 org.eclipse.cdt.core 4 0 2019-04-16 23:04:03.558
!MESSAGE IOException
!STACK 0
java.io.IOException: An unexpected network error occurred
	at sun.nio.ch.FileDispatcherImpl.pwrite0(Native Method)
	at sun.nio.ch.FileDispatcherImpl.pwrite(Unknown Source)
	at sun.nio.ch.IOUtil.writeFromNativeBuffer(Unknown Source)
	at sun.nio.ch.IOUtil.write(Unknown Source)
	at sun.nio.ch.FileChannelImpl.writeInternal(Unknown Source)
	at sun.nio.ch.FileChannelImpl.write(Unknown Source)
	at org.eclipse.cdt.internal.core.pdom.db.Database.write(Database.java:166)
	at org.eclipse.cdt.internal.core.pdom.db.Chunk.flush(Chunk.java:52)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flushAndUnlockChunks(Database.java:743)
	at org.eclipse.cdt.internal.core.pdom.db.Database.flush(Database.java:730)
	at org.eclipse.cdt.internal.core.pdom.PDOM.flush(PDOM.java:1418)
	at org.eclipse.cdt.internal.core.pdom.WritablePDOM.flush(WritablePDOM.java:192)
	at org.eclipse.cdt.internal.core.index.WritableCIndex.flush(WritableCIndex.java:165)
	at org.eclipse.cdt.internal.core.pdom.AbstractIndexerTask.runTask(AbstractIndexerTask.java:590)
	at org.eclipse.cdt.internal.core.pdom.indexer.PDOMIndexerTask.run(PDOMIndexerTask.java:146)
	at org.eclipse.cdt.internal.core.pdom.PDOMIndexerJob.run(PDOMIndexerJob.java:137)
	at org.eclipse.core.internal.jobs.Worker.run(Worker.java:53)

!ENTRY com.altera.sbtgui.launch 1 0 2019-04-16 23:04:59.118
!MESSAGE Executing: [C:/intelfpga/17.0/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=DE-SoC on localhost [USB-1]' --device=2 --instance=0 --sidp=0x1810 --id=0xad --timestamp=1555467743 /cygdrive/s/CPET-561/Arbitration_Demo/Part_1/HW/software/nios_system/nios_system.elf]

!ENTRY com.altera.sbtgui.launch 1 0 2019-04-16 23:05:03.797
!MESSAGE Executing: [C:/intelfpga/17.0/quartus\bin64\cygwin\bin\bash.exe, -c, nios2-download '--cable=DE-SoC on localhost [USB-1]' --device=2 --instance=0 --tcpport=auto --sidp=0x1810 --id=0xad --timestamp=1555467743]
